app/views/projects/delete.html.erb

```diff
@@ -0,0 +1,8 @@
+<div>
```

Comment:
You could avoid the need for this view by using `confirm` on the delete link, e.g. https://github.com/TechforgoodCAST/fusebox-insights/blob/master/app/views/unknowns/_card.html.erb#L28